tests/hmm/plotting.py

```diff
@@ -429,6 +429,27 @@ def create_variated_histos(proc,
             else:
                 hret = hdict[sname]
             ret[sname2] = hret
+ 
+    if('EWZ105160PS' in variations) and ('ewk_lljj_mll105_160_ptJ_herwig' in proc):
+        h_ps_pythia = copy.deepcopy(hdict_ps_pythia[baseline])
+        h_ps_herwig = copy.deepcopy(hdict_ps_herwig[baseline])
+        hnom = copy.deepcopy(hbase)
+        h_nom_up = copy.deepcopy(hbase)
+        h_nom_down = copy.deepcopy(hbase)
+        hnom = hnom * weight_xs[proc]
+        h_ps_pythia = h_ps_pythia * weight_xs[ps_pythia]
+        h_ps_herwig = h_ps_herwig * weight_xs[ps_herwig]
+        for k in range(len(hnom.contents)):
```

Comment:
You might be able to avoid the loop and just do the following:
```
h_nom_up.contents = hnom.contents + 0.2*(h_ps_pythia.contents - h_ps_herwig.contents)
```

Reply:
Thank you Joosep for the suggestion! I changed it.